lib/widgets/content.dart

```diff
@@ -940,7 +951,64 @@ class _KatexSpan extends StatelessWidget {
         textAlign: textAlign,
         child: widget);
     }
-    return widget;
+
+    if (styles.verticalAlignEm != null && styles.heightEm != null) {
+      assert(widget == defaultWidget);
```

Comment:
I think this assertion is based on the broader assumption that some inline styles never co-occur to `vertical-align` or `height`. This seems like something we can comment on.